src/components/TodoAddCard.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo,
+  isLoading: boolean,
+};
+
+export const TodoAddCard: React.FC<Props> = ({
```

Comment:
Is this a component for mocked todo to show it while it's adding?

If yes, than why can't we use usual TodoCard for this purpose? Or create component with mocked data and don't pass here mocked todo